app/src/display/widgets/label.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <zephyr/kernel.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+#include <zmk/display.h>
+#include <zmk/display/widgets/label.h>
+#include <zmk/events/activity_state_changed.h>
+#include <zmk/event_manager.h>
+
+static sys_slist_t widgets = SYS_SLIST_STATIC_INIT(&widgets);
+
+struct label_state {};
+
+static struct label_state get_state(const zmk_event_t *_eh) { return (struct label_state){}; }
+
+static void set_label_symbol(lv_obj_t *label) {
+    char text[7] = {};
+
+    strcat(text, CONFIG_ZMK_WIDGET_LABEL_TEXT);
+
```

Comment:
We should use a `BUILD_ASSERT` on the length here, and just directly call set_text with the constant value instead of using strcat here.

Reply:
Hey @petejohanson, thanks a lot for the feedback!

I was using strcat since I was playing around with values concats for fun, cleaned it up to a direct set,
also added the `BUILD_ASSERT` and updated branch to latest main(don't know if needed, did just to be sure).

Let me know if anything else is required!

Reply:
Cleanup pushed!